docs/guides/upgrade-v0.16.x.md

```diff
@@ -0,0 +1,188 @@
+# v0.16.x Upgrade Guide
+
+Version `0.16.x` brought a handful of [breaking, but really good changes](https://github.com/FormidableLabs/radium/blob/master/CHANGELOG.md#breaking-changes). Here's how you upgrade your app.
+
+## Keyframes and Media Queries with Server Side Rendering
+
+If you're using keyframes or media queries, you will need to wrap the top-level component in a `<StyleRoot/>` component.
+
+You can't use keyframes or media queries in the direct children of `<StyleRoot/>`. See the docs for more details: https://github.com/FormidableLabs/radium/tree/master/docs/api#styleroot-component
+
+### How to Make the Change
+
+Old syntax (`v0.15.x`)
+
+```jsx
+import Radium from 'radium';
+
+@Radium
+class App extends React.Component {
+  render() {
+    return (
+      <div>
+        ... rest of your app ...
+      </div>
+    );
+  }
+}
+```
+
+New syntax (`v0.16.x`)
+
+```jsx
+import {StyleRoot} from 'radium';
+
+// No need for @Radium decorator; StyleRoot is already wrapped.
+class App extends React.Component {
+  render() {
+    return (
+      <StyleRoot>
+        ... rest of your app ...
+      </StyleRoot>
+    );
+  }
+}
+```
+
+## Keyframes
+
+Wrap your root component in `<StyleRoot/>` (see above).
+
+The result of Radium.keyframes is the animation name, and should now be assigned to the animationName prop: https://github.com/FormidableLabs/radium/tree/master/docs/api#keyframes
```

Comment:
It used to be the actual animation name; now it is a placeholder object that Radium processes at render time.